#!/usr/bin/env python
PACKAGE = "contrail"

from dynamic_reconfigure.parameter_generator_catkin import *
import math

gen = ParameterGenerator()

gen.add("end_position_accuracy", double_t, 0, "Distance from the final point that counts as the trajectory being complete", 0.1, 0.0, None)
gen.add("end_yaw_accuracy", double_t, 0, "Yaw rotation from the the final point that counts as the trajectory being complete", 0.1, 0.0, 2*math.pi)
gen.add("spline_res_per_sec", int_t, 0, "Amount of points per second to use during the spline approximation feedback", 5, 0, None)
gen.add("output_accel_ref", bool_t, 0, "Enables additional acceleration reference to be added to the triplet", True)

exit(gen.generate(PACKAGE, "contrail", "ManagerParams"))
